MEMBER_TRAVEL_FUND.md

```diff
@@ -12,9 +12,11 @@ ecosystem and the Foundation.
 ## Requirements
 
 * Candidates must be an individual member of the Node.js Foundation. [The membership status can be checked on the foundation website](https://identity.linuxfoundation.org/user/login?destination=user/me). Due to privacy, the Individual Members list is not public. 
-This team will verify the requester is on the Individual Members list before funds are disbursed. 
+This team will verify the requester is on the Individual Members list before funds are disbursed.
+  * Exception: For the Collaborator Summits, members of JavaScript Foundation
```

Comment:
```suggestion
  * Exception: For the Collaborator Summits, members of JS Foundation
```

(I think they could not use the name JavaScript because it's trademarked, similar to why ECMAScript is ECMAScript)